src/components/footer/__tests__/__snapshots__/Footer.test.js.snap

```diff
@@ -3,35 +3,7 @@
 exports[`renders the footer element 1`] = `
 <footer>
   <div>
-    <div>
-      <form
-        action="https://www.paypal.com/cgi-bin/webscr"
-        method="post"
-        target="_top"
-      >
-        <input
-          type="hidden"
-          name="cmd"
-          value="_s-xclick"
-        />
-        <input
-          type="hidden"
-          name="hosted_button_id"
-          value="123abc"
-        />
-        <input
-          type="image"
-          src="https://www.paypalobjects.com/en_US/i/btn/btn_donateCC_LG.gif"
-          border="0"
-          name="submit"
-          alt="PayPal - The safer, easier way to pay online!"
-        />
-        <img
-          src="https://www.paypalobjects.com/de_DE/i/scr/pixel.gif"
-          alt="Donate!"
-        />
-      </form>
-    </div>
+    <div />
```

Comment:
Is it valid, according to the w3c spec, to have self closing divs?

Reply:
It's not valid since `html 4.x.` It's only valid if its `XHTML` if I am not wrong.

Reply:
We are using that div to pull the footer links to the right, we are temporarily hiding the donate button due to some issues with @emmawedekind 's paypal account. We will bring it back, I just don't want to refactor this, given that we are going to put it back soon.

Reply:
Ye but using `<div></div>` is probably a better idea, even if it's only temporary.

Reply:
This file is generated by Jest, we can't change it and is only for testing purposes, nothing to be deployed xD 

The actual html looks like this:

<img width="1005" alt="screen_shot_2018-10-30_at_1_58_02_pm" src="https://user-images.githubusercontent.com/219207/47743020-fe2fbf00-dc4b-11e8-9dde-93c05cbff587.png">

;)

Reply:
Wow I missed the file path (hence didn't realyze it's part of jest's snapshot testing), my bad sorry 🙈